build.gradle

```diff
@@ -39,9 +39,58 @@ jar {
 	enabled = false
 }
 
+task fatJavadoc(type: Javadoc) {
```

Comment:
Why make this separate from normal javadoc?

Reply:
i was having some issues with it being the normal javadoc, probably could move it back

Reply:
After testing, all that I can change are the names to match the standard. However, I feel like the names should stay the same as they are now, since it is not a normal javadoc generation.